odoo/addons/base/tests/test_misc.py

```diff
@@ -507,3 +507,71 @@ def test_readonly_dict(self):
             d.update({'baz': 'xyz'})
         with self.assertRaises(TypeError):
             dict.update(d, {'baz': 'xyz'})
+
+class TestFormatLang(TransactionCase):
+    # /!\ The monetary argument does not need to be tested on version <= 17.1 because it has no effect.
+    def test_value_and_digits(self):
+        self.assertEqual(misc.formatLang(self.env, 100.23, digits=1), '100.2')
+        self.assertEqual(misc.formatLang(self.env, 100.23, digits=3), '100.230')
+
+        # If value is an empty string, it should return an empty string (not 0).
```

Comment:
The comment seems perfect to be thrown in the console in case of error.
We should put it as the third argument of the assert method (and same for all the other)